Update Dockerfile

peetu opened the pull request:
npm is unaware where the python is (python3 is there), added this in order for anything that depends on python to work.

harbu commented on bin/publisher.js:
```diff
@@ -2,27 +2,37 @@
+    .option('--streamId <streamId>', 'streamId to publish', 'default-stream-id')
+    .option('--metrics <metrics>', 'log metrics', false)
+    .option('--intervalInMs <intervalInMs>', 'interval to publish in ms', 200)
+    .description('Run subscriber')
```
You mean `run publisher`?

peetu committed:
Update Dockerfile

npm is unaware where the python is (python3 is there), added this in order for anything that depends on python to work.

Commit:
Revert "github actions: broker.yml: remove arm64 for now"

This reverts commit 66e82c7d3157659358c7b1e2a2b24a2e3dd0b2e1.

Commit:
Run docker-push always (REVERT before merge)

harbu commented:
Hey I made a [commit](https://github.com/streamr-dev/network-monorepo/pull/349/commits/c95071a7c99d6a69a0c11e3190288af7dbbde1ef) to make the CI run `Docker Build + Push`. Let's see if that works now, and if it does, let's revert the commit before squash + merging.

juslesan committed:
Revert "Run docker-push always (REVERT before merge)"

This reverts commit c95071a7c99d6a69a0c11e3190288af7dbbde1ef.

juslesan commented:
> c95071a

This passed so I'm reverting and merging
